cratedb_sqlparse_py/cratedb_sqlparse/parser.py

```diff
@@ -132,8 +132,7 @@ class Metadata:
     table, schema, columns, options...
     """
 
-    schema: str = None
-    table_name: str = None
+    tables: list = dataclasses.field(default_factory=list)
```

Comment:
Suggestion to use `typing.List`, in order to stay compatible with Python 3.8.
```suggestion
    tables: List[Table] = dataclasses.field(default_factory=list)
```